src/Commands/SecretsCommand.php

```diff
@@ -202,7 +202,12 @@ protected function downloadSecrets($site_env_id, $filename)
     protected function uploadSecrets($site_env_id, $secretValues, $filename)
     {
         $workdir = $this->tempdir();
-        mkdir("$workdir/private");
+        $parent_folders = dirname($filename);
+        if ($parent_folders !== '.') {
+            mkdir("$workdir/private/$parent_folders", 0777, true);
```

Comment:
mkdir will only make at most one directory. If you use Symfony filesystem, then it will make the whole chain.

Reply:
hmmm, 3rd parameter is recursive so I think that's equivalent to `mkdir -p` Actually with this change, I tested this in my local env and it worked as expected 🤔

Reply:
Oh cool there's a third parameter of mkdir that I missed for over a decade. Carry on.